pkg/services/dashboards/service/integration_test/dashboard_k8s_test.go

```diff
@@ -0,0 +1,1680 @@
+				require.NoError(t, err)
+				require.NotNil(t, provisionedFetchedDash)
+
+				// Try to update the dashboard using editor (not admin)
+				_, err = updateDashboard(t, editorClient, provisionedFetchedDash, "Updated Provisioned Dashboard")
+
+				if tc.shouldSucceed {
+					require.NoError(t, err, "Editor should be able to update provisioned dashboard when allowsEdits is true")
+
+					// Verify the update succeeded by fetching the dashboard again
+					updatedDash, err := editorClient.Resource.Get(context.Background(), dashUID, v1.GetOptions{})
+					require.NoError(t, err)
+					updatedSpec := updatedDash.Object["spec"].(map[string]interface{})
+					require.Equal(t, "Updated Provisioned Dashboard", updatedSpec["title"], "Dashboard title should be updated")
+				} else {
+					require.Error(t, err, "Editor should not be able to update provisioned dashboard when allowsEdits is false")
+					require.Contains(t, err.Error(), "provisioned")
+				}
+
+				// Clean up
+				err = adminClient.Resource.Delete(context.Background(), dashUID, v1.DeleteOptions{})
+				require.NoError(t, err)
+			})
+		}
+	})
+
+	t.Run("Dashboard refresh interval validations", func(t *testing.T) {
+		t.Run("reject dashboard with refresh interval below minimum", func(t *testing.T) {
+			dashObj := createDashboardObject("Dashboard with Short Refresh", "", 0)
+
+			// Add refresh configuration that should be too frequent
+			if spec, ok := dashObj.Object["spec"].(map[string]interface{}); ok {
+				spec["refresh"] = "1s" // Too frequent, below typical minimum
+			}
+
+			_, err := adminClient.Resource.Create(context.Background(), dashObj, v1.CreateOptions{})
+			require.Error(t, err)
+		})
+	})
+
+	t.Run("Dashboard tag validations", func(t *testing.T) {
+		t.Run("reject dashboard with excessive number of tags", func(t *testing.T) {
+			// Create a dashboard with too many tags
+			dashObj := createDashboardObject("Dashboard with Too Many Tags", "", 0)
+
+			// Add excessive number of tags
+			tooManyTags := []string{}
+			for i := 0; i < 100; i++ { // Assuming 100 is more than the max limit
+				tooManyTags = append(tooManyTags, fmt.Sprintf("tag%d", i))
+			}
+
+			if spec, ok := dashObj.Object["spec"].(map[string]interface{}); ok {
+				spec["tags"] = tooManyTags
+			}
+
+			_, err := adminClient.Resource.Create(context.Background(), dashObj, v1.CreateOptions{})
+			require.Error(t, err)
+		})
+
+		t.Run("reject dashboard with invalid tag characters", func(t *testing.T) {
+			dashObj := createDashboardObject("Dashboard with Invalid Tags", "", 0)
+
+			// Add tags with invalid characters
+			invalidTags := []string{"tag with spaces", "tag/with/slashes", "tag,with,commas"}
+
+			if spec, ok := dashObj.Object["spec"].(map[string]interface{}); ok {
+				spec["tags"] = invalidTags
+			}
+
+			_, err := adminClient.Resource.Create(context.Background(), dashObj, v1.CreateOptions{})
+			require.Error(t, err)
+		})
+	})
+
+	t.Run("Dashboard panels validations", func(t *testing.T) {
```

Comment:
I can't find the panel validations in the dashboard service - where are you seeing those?

Reply:
These were some validations we had in the frontend. I've removed them